pkg/didcomm/protocol/didexchange/models.go

```diff
@@ -101,3 +91,126 @@ type Connection struct {
 	DID    string   `json:"did,omitempty"`
 	DIDDoc *did.Doc `json:"did_doc,omitempty"`
 }
+
+// ReceiveInvitationRequest model
+//
+// This model defines DID exchange receive invitation request
+//
+type ReceiveInvitationRequest struct {
```

Comment:
We already have `Invitation` for this

Reply:
This is not REST API model, REST API model uses this to wrap POST request (REST API wraps this model and adds open API annotations).
https://github.com/hyperledger/aries-framework-go/pull/194/files#diff-74a6baecd4124df46bc64072ff4c032eR38

I agree, I need to rename this model to avoid confusion, running out of ideas to give it a different name, any suggestions?

Reply:
Removed